src/admin/action/FetchOnlineDriverData.ts

```diff
@@ -1,22 +1,29 @@
-/* eslint-disable react-hooks/rules-of-hooks */
-import { useRepository } from "@/hooks/CustomHook";
-import { OnlineDriver } from "@/types/data";
-import _ from "lodash";
+import { useEffect, useState } from 'react';
```

Comment:
Aren't these throwing errors? "useEffect" and "useState" are hooks and can't be used here. Just return the response data received.